src/test/java/gov/healthit/chpl/aqa/stepDefinitions/LoginSteps.java

```diff
@@ -21,6 +21,8 @@
     private String url = System.getProperty("url");
     private String username = System.getProperty("username");
     private String password = System.getProperty("password");
+    private String adminUsername = System.getProperty("adminUsername");
+    private String adminPassword = System.getProperty("adminPassword");
```

Comment:
All of our users are Admin in one way of another. If you're going to distinguish between them, use the ROLEs we use everywhere else. To get all three of the AQA users in this property file you should have
```
roleAdminUsername
roleAdminPassword
roleOncUsername
roleOncPassword
roleAcbUsername
roleAcbPassword